index.js

```diff
@@ -36,16 +35,17 @@ a WebExtension from the command line:
   ${chalk.bold.blue("web-ext run -s /path/to/extension")}
 `;
 
+const asciiLogo = fs.readFileSync(
```

Comment:
uhm... I'm particularly not thrilled about adding an `fs.readFileSync` call here, especially considering about this module used as a library, it basically means that the nodejs app that includes this module is going to "pause everything else" until it has loaded the asciiLogo.

is exporting the asciiLogo actually needed? we can discuss about why we need it and find another solution for it.

Reply:
@rpl I agree this is not ideal, i decided to do that because `__dirname` is different for tests, but now I just pass correct directory.